patch/vanitygaps.c

```diff
@@ -86,6 +86,16 @@ togglegaps(const Arg *arg)
 	#else
 	enablegaps = !enablegaps;
 	#endif // PERTAG_VANITYGAPS_PATCH
+
+	#if BAR_GAPS_PADDING_PATCH
+	updatebarpos(selmon);
+	Bar *bar = selmon->bar;
+	XMoveResizeWindow(dpy, bar->win, bar->bx, bar->by, bar->bw + gappoh, bar->bh);
```

Comment:
I think we should probably loop through all the bars that the monitor has present, rather than just changing the position of the first one.

Reply:
Done!